src/Security/Authentication/MicrosoftAccount/src/MicrosoftAccountOptions.cs

```diff
@@ -21,6 +21,7 @@ public MicrosoftAccountOptions()
             AuthorizationEndpoint = MicrosoftAccountDefaults.AuthorizationEndpoint;
             TokenEndpoint = MicrosoftAccountDefaults.TokenEndpoint;
             UserInformationEndpoint = MicrosoftAccountDefaults.UserInformationEndpoint;
+            UsePkse = true;
```

Comment:
Is this supposed to be PKCE?  (Just going off title of issue)

Reply:
Yes 😁